### PR TITLE
Updated Styling

### PR DESCRIPTION
**Pull request by aryamanpuri**
**WORK DONE** 
1.Fixed Footer
2. Changed the feedback form to a new page
3. Fixed the `See our Search Syntax Guide here ` on the landing page which get affected on resizing the window.
image

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by aryamanpuri** [comment]
Fixing the footer will lead to undesirable results, so I am removing it from footer

File: src/pages/FeedbackPage.vue [hunk 1]
```diff
@@ -10,19 +10,9 @@
         <a href="https://wiki.creativecommons.org/wiki/Slack#How_to_join_Slack">CC Slack</a>.
       </p>
       <p>
-        To report a bug, <a href="#report">click here</a>.
+        To give Feedback, <a href="https://docs.google.com/forms/d/e/1FAIpQLSfb_6yq2Md0v6S-XzsyT7p1QVhqr7MWHqInKdyYh4ReaWn4FQ/viewform?embedded=true" target="_blank">click here</a>.
```
Comment (brenoferreira) [inline src/pages/FeedbackPage.vue]: Please undo these changes. These are separate things and we need the Google Doc iframe

Comment (aryamanpuri) [reply src/pages/FeedbackPage.vue]: @brenoferreira okay

Comment (aryamanpuri) [reply src/pages/FeedbackPage.vue]: @brenoferreira But making google form to another tab will make it a much better view wise
